draft-ietf-quic-http.md

```diff
@@ -1165,10 +1165,11 @@ desired. They MAY also be sent on connections where no data is currently being
 transferred. Endpoints MUST NOT consider these streams to have any meaning upon
 receipt.
 
-The payload and length of the stream are selected in any manner the
-implementation chooses.  Implementations MAY terminate these streams cleanly, or
-MAY abruptly terminate them.  When terminating abruptly, the error code
-H3_NO_ERROR or a reserved error code ({{http-error-codes}}) SHOULD be used.
+The payload and length of the stream are selected in any manner the sending
+implementation chooses.  When sending a reserved stream type, the implementation
+MAY either terminate the stream cleanly or reset it.  When resetting the stream,
```

Comment:
```suggestion
MAY either allow the stream to terminate cleanly or reset it.  When resetting the stream,
```

Reply:
On the sender side, though, either is an explicit action; it's not something you just allow to happen.

Reply:
@martinthomson : Is your point that terminating cleanly is application action where resetting is transport action?

Reply:
As a receiver you have to let the stream terminate cleanly, the only other action available is a stream reset.

Presumably the sender knows what they are doing, so I wanted to concentrate on the reaction the receiver might have.

Reply:
The generic advice we give for unknown stream types (which this should be) is to reply with a STOP_SENDING.  On the receiver side, you shouldn't be identifying these as grease streams.

Reply:
This is entirely about what the sender does, right?